Ag manufacturing patches (#315)
Azure/arc_jumpstart_docs: docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md

---
type: docs
weight: 5
title: Enabling AI at the Edge to enhance workers safety
linkTitle: Enabling AI at the Edge to enhance workers safety
summary: |

>     The enabling AI at the Edge to enhance workers safety page provides an overview of how Contoso Motors leverages AI to ensure workers' safety by detecting workers with no helmets on the factory floor. It describes the architecture and flow of information for detecting and classifying helmet adherence using AI. The page also explains the steps involved in the inference process, including UI selection, RTSP video simulation, frame capturing, image pre-processing/inferencing, and post-processing/rendering.

Reviewer: ```suggestion
    The enabling AI at the Edge to enhance workers safety page provides an overview of how Contoso Motors uses AI to ensure workers' safety by detecting workers with no helmets on the factory floor. It describes the architecture and flow of information for detecting and classifying helmet adherence using AI. The page also explains the steps involved in the inference process, including UI selection, RTSP video simulation, frame capturing, image pre-processing/inferencing, and post-processing/rendering.
```

serviceOrPlatform: Manufacturing
technologyStack:
  - AKS
  - OPENVINO
  - AI
  - AKS EDGE ESSENTIALS
  - RTSP
---

# Enabling AI at the Edge to enhance workers safety

## Overview

> Contoso Motors uses AI-enhanced computer vision to improve workers' safety by detecting workers with no helmets on the factory floor. Workers safety is one of the four computer vision use cases that Contoso Motors uses, which also include object detection, and human pose estimation. While each use case has its own unique characteristics, they all follow the same inferencing architecture pattern and data flow.

Reviewer: ```suggestion
Contoso Motors uses AI-enhanced computer vision to improve workers' safety by detecting workers with no helmets on the factory floor. Worker safety is one of the four computer vision use cases that Contoso Motors uses, which also include object detection, defect detection, and human pose estimation. While each use case has its own unique characteristics, they all follow the same inferencing architecture pattern and data flow.
```

## Architecture

> ![Welding defect archietcture](./img/flow.png)

Reviewer: ```suggestion
![Welding defect architecture](./img/flow.png)
```

### Model

#### Inputs

#### Outputs

## Operation technology (OT) Manager Experience

> Contoso leverages their AI-enhanced computer vision to monitor the safety helmet adherence for workers on the factory floor to help OT managers ensure workers safety through the "Control Center" interface.

Reviewer: ```suggestion
Contoso uses AI-enhanced computer vision to monitor the safety helmet adherence for workers on the factory floor to help OT managers ensure workers safety through the "Control Center" interface.
```
